src/openforms/js/public.js

```diff
@@ -17,3 +18,35 @@ const init = () => {
 };
 
 onLoaded(init);
+
+// observe whether cookiebanner is in sticky mode or not
+// note: rootMargin must be specified in pixels or percent, not REMs/EMs.
+const observer = new IntersectionObserver(
+  ([e]) => e.target.classList.toggle('stuck', e.intersectionRatio < 1),
+  {root: document.querySelector('body'), rootMargin: '0px 0px 31px 0px', threshold: [1]}
+);
+
+observer.observe(document.querySelector('.cookie-notice'));
+
```

Comment:
@sergei-maertens I give up 😟; this code worked shortly (where the drop-shadow would disappear neatly when scrolled down) but now it is not working anymore - in this same public.js file I've done some more experimenting (with different results in desktops and mobile). Something is conflicting somewhere, and I'm not sure what it might be? Also: the Master/Base html templates may not be correct.